pkg/controller/publisher.go

```diff
@@ -38,16 +38,16 @@ type VolumePublisher interface {
 	// CheckIfVolumeExists queries storage array if given volume already exists
 	CheckIfVolumeExists(ctx context.Context, client gopowerstore.Client, volID string) error
 	// Publish does the steps necessary for volume to be available on the node
-	Publish(ctx context.Context, req *csi.ControllerPublishVolumeRequest, client gopowerstore.Client,
-		kubeNodeID string, volumeID string) (*csi.ControllerPublishVolumeResponse, error)
+	Publish(ctx context.Context, publishContext map[string]string, req *csi.ControllerPublishVolumeRequest, client gopowerstore.Client,
```

Comment:
Will this change impact other users of this interface outside of csi-powerstore?

Reply:
No. It seems to be powerstore driver specific.